File: src/main/java/org/polyfrost/glintcolorizer/command/GlintCommand.java
package org.polyfrost.glintcolorizer.command;

//#if MODERN==0

import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
import cc.polyfrost.oneconfig.utils.commands.annotations.Main;
import org.polyfrost.glintcolorizer.GlintColorizer;

@Command(value = "glintcolorizer", description = "Glint Colorizer commands")
public class GlintCommand {

    @Main
    public void handle() {
        GlintColorizer.config.openGui();
    }
}
//#endif

File: src/main/kotlin/org/polyfrost/glintcolorizer/command/GlintCommand.kt
package org.polyfrost.glintcolorizer.command

import org.polyfrost.glintcolorizer.GlintColorizer
import org.polyfrost.glintcolorizer.config.GlintConfig
import org.polyfrost.oneconfig.api.commands.v1.factories.annotated.Command
import org.polyfrost.oneconfig.utils.v1.dsl.openUI

@Command(
    value = [GlintColorizer.ID],
    description = "Access the " + GlintColorizer.NAME + " GUI."
)
class GlintCommand {

    @Command
    fun handle() {
        GlintConfig.openUI()
    }

}